helm_chart/recordingbot/values.yaml

```diff
@@ -0,0 +1,18 @@
+# global:
+app:
+  replica: 3
+  image: quay.io/asagtani_wire/recordingbot
+  tag: latest
+  env:
+    - name: SERVICE_TOKEN
+      value: # Provide the Recording bot authentication token here
+    - name: PUBLIC_URL
```

Comment:
`EXTERNAL_URL` seems to be a more common name for this. can you make this configurable via a `config.externalUrl` helm value?